package.json

```diff
@@ -5,18 +5,20 @@
   "private": true,
   "scripts": {
     "clean:client": "cd ./client/ && rm -rf ./node_modules && npm install --only=production",
-    "clean:server": "cd ./server/ && rm -rf ./node_modules && npm install --only=production && cd node_modules && echo num dep files before prune: $(find . -type f | wc -l) && find . -type f -not \\( -name '*.js' -or -name '*.json' -or -name '*.node' \\) -delete && echo num dep files after prune: $(find . -type f | wc -l)",
+    "clean:server": "cd ./server/ && rm -rf ./node_modules && npm install --only=production && cd - && npm run build-prod-grpc-extension && npm run prune-server-deps",
+    "build-prod-grpc-extension": "cd ./server/node_modules/grpc && node_modules/.bin/node-pre-gyp install --fallback-to-build --library=static_library --target_arch=x64 --target_platform=linux --runtime=node --target=8.9.0 --target_libc=glibc",
```

Comment:
This is kinda crazy. Maybe a comment would be helpful?

https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install

Reply:
Huh, didn't know that was valid. Added a comment for this and the `prune-server-deps` command.

Reply:
Thanks. It's a bit awkward, but in this case I think it needed a comment.